workflows/deploy-cloudrun/service.template.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: $SERVICE
```

The Google style guide suggests always using the curly-brace format:

```suggestion
  name: ${SERVICE}
```

---

Thank you for pointing that out... resolved it